script/bigscreenplayer.js

```diff
@@ -253,6 +265,9 @@ define('bigscreenplayer/bigscreenplayer',
         },
         setSubtitlesEnabled: function (value) {
           playerComponent.setSubtitlesEnabled(value);
+          subtitleCallbacks.forEach(function (callback) {
```

Comment:
Something that just came to mind. You might consider implementing this in `playerComponent` due to the `enableSubtitles` flag that is passed into `playerComponent` when it is first used. I don't think this would get the initial state emitted.

Reply:
good catch. I pushed a commit for this. It keeps it in this file for now because this is where the other event like callbacks are  but can move it into the player component if you want.